AEPCore/Tests/MobileCoreTests.swift

```diff
@@ -187,29 +187,29 @@ class MobileCoreTests: XCTestCase {
 
         let expected = """
         {
-          "extensions" : {
```

Comment:
Keeping this top level key around might be useful if we need to add other information (new keys) in this payload in the future.